packages/core/src/utils/rateLimiting/types.ts

```diff
@@ -0,0 +1,16 @@
+type Unit = "ms" | "s" | "m" | "h" | "d";
+export type Duration = `${number} ${Unit}` | `${number}${Unit}`;
```

Comment:
Is there any danger in this being called simply Duration? Eg. for autocomplete / auto-import?

Reply:
Eg. RateLimitDuration, RateLimitUnit

Reply:
I don't know about danger, but I agree it could be annoying. I've renamed it to RateLimitDuration